tests/test_websocket.py

```diff
@@ -52,15 +51,16 @@ def test_websocket(self):
         self.assertEqual(len(self.base_driver
                                  .find_elements_by_class_name('leaflet-marker-icon')), 0)
         # create marker on another window
-        self.log_in(self.change_location_driver,
-                    self.config['app_url'], self.config['username'], self.config['password'])
+        self.log_in(self.change_location_driver, self.config['app_url'],
+                    self.config['username'], self.config['password'])
         self.get_location(self.change_location_driver,
                           self.config['app_url'], self.location_name)
         self.set_location_marker(self.change_location_driver,
                                  self.location_name)
         # assert that marker popped up on map
         self.assertEqual(len(self.base_driver
                                  .find_elements_by_class_name('leaflet-marker-icon')), 1)
+        print('SUCCESS: Websocket service is working!')
```

Comment:
Print is not required. 
When this branch is merged to master the shell script will be removed and we will run tests like in other repositories! :smile: